### ISSUE TITLE
Could we please have a project status report please?

### ISSUE THREAD

**Issue by JohnnyC1951** [LibreTime/libretime#488]
It would appear that development is stalled at the moment. 

I am sure that if the devs are too busy, others would be happy to help. Some of us are rather reliant upon this project and are getting worried.

**Comment by Robbt** [comment]
Sorry to hear about your companies decision. LibreTime is definitely a work in progress and at times that progress may seem stifled by a lack of full-time volunteers. I know you poured your heart and head into building a solution for your projects based upon the amount of hacking you did. Whatever code you want to share would be appreciated. I think in theory according to the AGPLv3 even commercially modified code behind hosted services is supposed to be shared but I'm not exactly a copyright lawyer and just know that is the legal theory behind AGPLv3 which SourceFabric switched Airtime too before we forked. But anyways I'll continue doing what I can. 

In true honesty I think that LibreTime is at a sort of impasse because the base upon which it is built ZendPHP is at EOL, which makes supporting it moving forward more difficult. There are all kinds of goofy issues that emerge due to the liquidsoap versions and python versioning that take place and we don't have anyone working on it full-time in a commercial capacity, especially now that your company decided to shut down its usage of it.

What I'd like to see is for us to clean up what we have and polish it into a legacy installable version that works out of the box w/ specific distros and well supported documentation and then start refactoring with a newer base. Who if anyone is going to do this work remains to be seen. I documented a whole slew of issues with our current documentation and nobody including myself has spent the time to fix them. So it may just be that we are challenged by a lack of forward momentum as a community and a bit of absenteeism by the project leaders. I'm going to try to keep a positive attitude towards it because sowing discontent and what not on the Internet is generally a waste of time in my experiences.

**Comment by JohnnyC1951** [comment]
Any mods I did to to the LT core I have published and shared in this Git and or , but has not been adopted, so that is that part already dealt with. As the rest of the developed code is all stand alone, cron based and does not patch or modify the LT code at all, and works even in the absence of any radio station software, I do not see how LT or AT could possibly have any rights to it. However, I may de-brand, pack it all into an appliance image and cast it into the public domain as a turnkey solution and / or stuff it in a GIT, when I have a spare weekend.
